Pipeline/pipeline.py

```diff
@@ -1,51 +1,39 @@
-# -*- coding: utf-8 -*-
-"""
-Created on Tue Jul 21 12:10:24 2020
+"""ROSS Machine Learning.
 
-@author: Luan
+ROSS-ML is a module to create neural networks aimed at calculating rotodynamic
+coefficients for bearings and seals.

```

Comment:
1) Divide data in two groups: labels and features. After dividing it, choose a sample to cross-validate the data.
2 - question 1) Please rephrase your question. What exactly do you mean by assumptions? 
2 - question 2) It depends. For classification problems, sigmoids are prefered on output. For regression, you should use ReLU, or any other. The layer will affect the topology to optmize the loss function
3) Cross-validation gives you a set to be compared with the trained one. In general, you should use a metric according to your problem. It varies a lot. For regression, you could use R^2,R^2 adjusted,MAE or MSE. For classification, cross-entropy is prefered. Here I also added KS test. The idea between KS test is that the original data and trained data should arise from the same distribution if the train succeded

Reply:
First question with more details:

"How are we supposed to train the neural network - with data from other models, CFD results or test data?"

- 
You can use any type of data as long as it is possible to convert it to a structured DataFrame form in which you can identify the  exogenous variables (inputs) and endogenous ones (output). The main idea of pipeline is scalability. To sum up: you must look to your problem and make the following questions: 1)"Can the problem  be converted into structured data?" 2)"Is it possible to divide it clearly in inputs and outputs?"

Reply:
> Divide data in two groups: labels and features. After dividing it, choose a sample to cross-validate the data.
2 - question 1) Please rephrase your question. What exactly do you mean by assumptions?
2 - question 2) It depends. For classification problems, sigmoids are prefered on output. For regression, you should use ReLU, or any other. The layer will affect the topology to optmize the loss function

I mean that, as a user, I would like to know the basics of how the code inside the 'Pipeline' class works, so layers type was just an example of information that I would like to see here at the documentation. 

> Cross-validation gives you a set to be compared with the trained one. In general, you should use a metric according to your problem. 

For our specific problem (taking the seal as an example), do we have any recommendation for how the user should carry out the validation?

Reply:
Yes, we have. The first metric used in validation is R squared. when it is close to one, the odds of obtaining a good result are high. The R^2 must be used between the chunk of data used to cross validation and the trained data. In a ideal world, trained data will be equal tested data and, as a result, R^2 = 1, which tells us that tested data = trained data. When you avail the problem by the means of R^2, you are asking to your model how close the trained set are closed to the line tested data = trained data. This kind of result are also shown in a q-q plot graph

Reply:
If you want a notion of distance, you should use the Mean Squared Error, Mean Absolute error or anything with the same concept. The main idea here is to measure how far your points are from its refferences in average. MSE punishes hard the furthest points, while its very soft to the closest ones. Note that your scale is important: use MSE on data with order of 1e-1 it is useless. MAE have the same idea as MSE, but treat only the distance between the points in modulus, instead of punishing the furthest ones by squaring it all the points. The metric used will depend strongly on which typing of problem you are facing with

Reply:
"I mean that, as a user, I would like to know the basics of how the code inside the 'Pipeline' class works, so layers type was just an example of information that I would like to see here at the documentation."

Fair enough. I will try to answer all these kind of question in a robust README, with all the required information to use properly pipeline without any major issues.

Reply:
`Yes, we have. The first metric used in validation is R squared. when it is close to one, the odds of obtaining a good result are high. The R^2 must be used between the chunk of data used to cross validation and the trained data. In a ideal world, trained data will be equal tested data and, as a result, R^2 = 1, which tells us that tested data = trained data. When you avail the problem by the means of R^2, you are asking to your model how close the trained set are closed to the line tested data = trained data. This kind of result are also shown in a q-q plot graph`

Nice explanation! I was thinking of having something like this in our docs, always focusing on our problem type, which will always be something like the one we had for the seal: some input such as inlet_temperature, inlet_pressure, radius, etc. and output kxx, kxy, kyx, kyy,  cxx, cxy, cyx, cyy.